app/views/organisation_reports/without_users_index.html.erb

```diff
@@ -1,3 +1,7 @@
+<p><b> The Following email will be sent when a user is invited: </b></p>
+<p>
+  <%= render template: 'custom_devise_mailer/invitation_instructions', :locals => { :@resource => User.new } %>
```

Comment:
looking at this again I think we probably want the following:

```ruby
<%= render template: 'custom_devise_mailer/invitation_instructions', :locals => { :resource => @user } %>
```
we shouldn't name a local variable with an at symbol, and we should prefer to set User.new in the controller.  Although all this could be replaced by some of those gems we were looking at.  Anyway ...